MoneyAPP/moneyapp/lib/blocs/transactions/transactions_bloc.dart

```diff
@@ -1,14 +1,17 @@
+import 'dart:convert';
+
 import 'package:bloc/bloc.dart';
 import '../../source/api_source.dart';
 import '../../models/transaction_model.dart';
 import 'transactions_event.dart';
-import 'package:logger/logger.dart';
 import 'transactions_state.dart';
+import 'package:logger/logger.dart';
 
 class TransactionsBloc extends Bloc<TransactionsEvent, TransactionsState> {
```

Comment:
Convert to Cubit, we don't really need Bloc as we are not doing any event transfomations. After that transformation, implement https://pub.dev/packages/value_cubit, and wrap the body of `void ApplyForLoan()` function with perform, so it correctly handles errors.

Reply:
Fixed